### ISSUE TITLE
voxCeleb1 and libri speech

### ISSUE THREAD

**Issue by hdubey** [mravanelli/SincNet#25]
Hi Mirco,
Thanks for the great work! I was wondering if you plan to share the data preparation recipe for voxCeleb1 and librispeech that can allow us to reproduce other experiments from your paper.

**Comment by mravanelli** [comment]
Hi,
the librispeech data used in my paper are available upon requests (I can
share with you if you want). We plan to move the speaker id experiments
within the pytorch-kaldi project in the near future. The full pipelines
starting from data creation to performance evalutation will be available
there for several datasets..

On Apr 1, 2019 18:22, "hdubey" <notifications@github.com> wrote:

Hi Mirco,
Thanks for the great work! I was wondering if you plan to share the data
preparation recipe for voxCeleb1 and librispeech that can allow us to
reproduce other experiments from your paper.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/mravanelli/SincNet/issues/25>, or mute the thread

**Comment by hdubey** [comment]
That would be helpful for me. Can you please share it on hxd150830@utdallas.edu

Thanks!

**Comment by mravanelli** [comment]
Ok,
this will likely happen in the next couple of months...there is a lot of
work to export everything to pytorch-kaldi

**Comment by hdubey** [comment]
Hi,
Thanks for replying. I just noticed your Librispeech.cfg has 8kHz sampling rate while TIMIT is 16kHz. Librispeech is avaliable at 16kHz, why did you downsampled those ? For a good comparison, both datasets should be at same Fs as per the general audio processing conventions. Any good reasons for doing so. Did you get better results at 8kHz for librispeech?

**Comment by mravanelli** [comment]
The difference in terms of performance is not that much between 8 kHz and
16 kHz for a speaker-id task.  For librispech, I finally saw a slightly
better performance at 8kHz and the network is also much faster.

Mirco

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by hdubey** [comment]
Thanks. It would be good to have these numbers into your Readme as most experiments take few to several days. I see most .cfg files has 0 drop-out factor meaning no drop-out. Is it the best configuration or just default. What drop-out you used for best results in arxiv paper ? Thanks for your time Mirco.

**Comment by mravanelli** [comment]
In the repository, we release the best system that we were able to find so
far. It is slightly better than that reported in the paper, because we
worked on this task also after the deadline and we found a set of the
hyperparameters a bit better.  So consider the cfg in the repo the best
config file found so far.

Mirco

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by hdubey** [comment]
Sounds good. Did you experimented with window length and skip rate. Do you think 200ms windows are better than 1second windows. Is it true that 10ms skip rate was better than 50ms skip on 1second windows. I am looking to speed up the training with least degradation. Since drop-out did not helped, other than lower sampling rate, what could help in accelerating the training process?

**Comment by hdubey** [comment]
I see that when the job crash due to memory or other issues, it do not resume from previous epoch. What change in code is needed to resume training from previous epoch without causing any degradation from uninterrupted training? Thanks for all support, we all like SincNet and trying to push its capability in all audio applications.

**Comment by mravanelli** [comment]
Maybe you can try with 15 or 25 ms skip and see if the degradation is
acceptable. You can also try windows if 100/150 ms that works quite well.

**Comment by Aki58** [comment]
hi,
can i get the librispeech data used in paper and all data preparation files (train.scp,test.scp,labels.npy).
at akashsingh24695@gmail.com

**Comment by mravanelli** [comment]
Hi Akash,
sure. At the link below you can find the dataset used for our speaker
recognition experiments:

https://drive.google.com/open?id=1lDHRUIWzZDg_0fauo3zh4VA8LgqkwsLJ

Best,

Mirco

**Comment by hdubey** [comment]
@mravanelli are there any filelists for reproducing the voxCeleb and voxCeleb2 experiments? Thanks!

**Comment by mravanelli** [comment]
For VoxCeleb we just used the standard split provided withing the VoxCeleb
dataset..

**Comment by Liujingxiu23** [comment]
@mravanelli  Have you  tried the proformace of sincnet in voxceleb ? what about the eer?
